common/lib/xmodule/xmodule/js/spec/capa/display_spec.coffee

```diff
@@ -688,19 +692,21 @@ describe 'Problem', ->
         $('#answer_1_1').html('One')
         $('#answer_1_2').html('Two')
 
-      it 'hide the answers', ->
-        @problem.show()
-        expect($('#answer_1_1')).toHaveHtml ''
-        expect($('#answer_1_2')).toHaveHtml ''
-        expect($('label[for="input_1_1_1"]')).not.toHaveAttr 'correct_answer'
+  describe 'displayShowAnswerTooltip', ->
+    beforeEach ->
+      @problem = new Problem($('.xblock-student_view'))
+      @problem.el.find('.show').isMouseFocus = false
 
-      it 'toggle the show answer button', ->
-        @problem.show()
-        expect($('.show .show-label')).toHaveText 'Show Answer'
+    it 'shows the tooltip', ->
+      expect(@problem.el.find('.show-answer-tooltip')).not.toHaveClass 'sr'
 
-      it 'remove the showed class from element', ->
-        @problem.show()
-        expect(@problem.el).not.toHaveClass 'showed'
+  describe 'hideShowAnswerTooltip', ->
+    beforeEach ->
+      @problem = new Problem($('.xblock-student_view'))
+      @problem.el.find('.show').isMouseFocus = true
```

Comment:
Is this really showing and hiding the tooltip, or is it just that the tooltip is never shown (and thus has the sr class because that is the default state)?